vivid-client/src/views/game/Draw.ts

```diff
@@ -102,3 +143,57 @@ function net(canvas: HTMLCanvasElement, context: CanvasRenderingContext2D) {
     context.fillRect(net.x, net.y + i, net.width, net.height);
   }
 }
+
+function playerZeroSpecial(
+  gameState: IGameState,
+  canvas: HTMLCanvasElement,
+  context: CanvasRenderingContext2D,
+) {
+  context.fillStyle = 'RED';
+  context.fillRect(
+    (gameState.players[0].x + (gameState.playerWidth / 3) * 2) * canvas.width,
+    gameState.players[0].y * canvas.height,
+    (gameState.playerWidth / 3) * canvas.width,
+    gameState.playerHeight * canvas.height,
+  );
+  context.fillStyle = 'YELLOW';
+  context.fillRect(
+    gameState.players[0].x * canvas.width,
+    gameState.players[0].y * canvas.height,
+    gameState.playerWidth * canvas.width,
+    (gameState.playerHeight / 5) * canvas.height,
+  );
+  context.fillRect(
+    gameState.players[0].x * canvas.width,
+    (gameState.players[0].y + (gameState.playerHeight / 5) * 4) * canvas.height,
+    gameState.playerWidth * canvas.width,
+    (gameState.playerHeight / 5) * canvas.height,
+  );
+}
+
+function playerOneSpecial(
+  gameState: IGameState,
+  canvas: HTMLCanvasElement,
+  context: CanvasRenderingContext2D,
+) {
+  context.fillStyle = 'RED';
+  context.fillRect(
+    gameState.players[1].x * canvas.width,
+    gameState.players[1].y * canvas.height,
+    (gameState.playerWidth / 3) * canvas.width,
+    gameState.playerHeight * canvas.height,
+  );
+  context.fillStyle = 'YELLOW';
+  context.fillRect(
+    gameState.players[1].x * canvas.width,
+    gameState.players[1].y * canvas.height,
+    gameState.playerWidth * canvas.width,
+    (gameState.playerHeight / 5) * canvas.height,
+  );
+  context.fillRect(
+    gameState.players[1].x * canvas.width,
+    (gameState.players[1].y + (gameState.playerHeight / 5) * 4) * canvas.height,
+    gameState.playerWidth * canvas.width,
+    (gameState.playerHeight / 5) * canvas.height,
+  );
```

Comment:
Are these exactly the same? Maybe use one function that takes a playernumber?

Reply:
good catch, im gonna merge it and fix it in my branch.